src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -2152,24 +3305,314 @@ namespace Google.Example.Library.V1.Samples
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generated code. DO NOT EDIT!
+
+// This is a generated sample ("Request", "test_default_calling_form_unary")
+
+// sample-metadata
+//   title: test_default_calling_form_unary
+//   description: Test default calling forms for unary methods.
+//   usage: dotnet run
+
+using CommandLine;
+
+// [START sample_get_shelf]
+namespace Google.Example.Library.V1.Samples
+{
+    using Google.Example.Library.V1;
+    using System;
+
+    public class SampleGetShelf
+    {
+        /// <summary>
+        /// Test default calling forms for unary methods.
+        /// </summary>
+        public static void SampleGetShelf()
+        {
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            GetShelfRequest request = new GetShelfRequest
+            {
+                ShelfName = new ShelfName("my-shelf"),
+                Options = "",
+            };
+            Shelf response = libraryServiceClient.GetShelf(request);
+            Console.WriteLine($"The theme of the shelf is: {response.Theme}");
+        }
+        // [END sample_get_shelf]
+        public static void Main(string[] args)
+        {
+            SampleGetShelf();
+        }
+    }
+}
+============== file: Google.Example.Library.V1/Google.Example.Library.V1.Samples/StreamShelvesFlattenedStreamingBidiEmpty.cs ==============
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generated code. DO NOT EDIT!
+
+// This is a generated sample ("FlattenedStreamingBidi", "empty")
+
+// sample-metadata
+//   title: Server streaming
+//   description: Testing calling forms
+//   usage: dotnet run
+
+using CommandLine;
+
+// [START sample]
+namespace Google.Example.Library.V1.Samples
+{
+    using Google.Example.Library.V1;
+    using System;
+
+    public class StreamShelvesFlattenedStreamingBidiEmpty
+    {
+        /// <summary>
+        /// Testing calling forms
+        /// </summary>
+        public static async Task SampleStreamShelvesAsync()
+        {
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
```

Comment:
This is an async method, so use `CreateAsync()`

Reply:
Copied from the Snippets which are using `Create()`: https://github.com/googleapis/gapic-generator/blob/11332f06a6ee95818412dcb9a715d7f87f1a8520/src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline#L4375

What's the difference here?